src/overview/actions.js

```diff
@@ -9,6 +9,8 @@ import { ourState } from './selectors'
 
 export const setQuery = createAction('overview/setQuery')
 export const setSearchResult = createAction('overview/setSearchResult')
+export const incrementLoadCounter = createAction('overview/incrementLoadCounter')
+export const decrementLoadCounter = createAction('overview/decrementLoadCounter')
```

Comment:
Due to name collision may I change it to showLoadIndicator and hideLoadIndicator?